jenkins/upgrade_python_requirements.py

```diff
@@ -57,14 +56,21 @@ def main(sha, repo_root):
                 user.name
             )
             create_branch(repository, branch, commit_sha)
+
             logger.info("Checking if there's any old pull requests to delete")
-            close_existing_pull_requests(repository, user.login, user.name)
+            deleted_pulls = close_existing_pull_requests(repository, user.login, user.name)
+
+            pr_body = "@edx/testeng please review"
+            for num, deleted_pull_number in enumerate(deleted_pulls):
+                if num == 0:
+                    pr_body += "\n\nDeleted obsolete pull_requests:"
+                pr_body += "\nhttps://github.com/edx/{}/pull/{}".format(repository_name, deleted_pull_number)
```

Comment:
I suspect that at some point we'll want to also run this for some repositories under edx-solutions, edx-it, edx-ops, etc.; not sure if you want to account for that now or save it for a later refactoring when the need arises.

Reply:
No time like the present!